app/models/IncomeFromWorkDependentStates.scala

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+import models.TagStatus.{CannotStartYet, Completed, NotStarted}
+
+case class IncomeFromWorkDependentStates(aboutYouSection: Boolean, aboutYourWorkPage: Boolean, jsaEsaPage: Boolean) {
```

Comment:
Just wondering would it better to call this class 
```suggestion
case class TailoringJourneySectionStates(aboutYouSection: Boolean, aboutYourWorkPage: Boolean, jsaEsaPage: Boolean) {
```

Reply:
will revist this later, as right now it is specific to one journey